polynote-spark/src/main/scala/polynote/kernel/interpreter/python/PySparkInterpreter.scala

```diff
@@ -30,14 +30,15 @@ class PySparkInterpreter(
 
   val gatewayRef = new AtomicReference[GatewayServer]()
 
-  override protected def injectGlobals(globals: PyObject): RIO[CurrentRuntime, Unit] = jep {
-    jep =>
-      val setItem = globals.getAttr("__setitem__", classOf[PyCallable])
+  override protected def injectGlobals(globals: PyObject): RIO[CurrentRuntime, Unit] = super.injectGlobals(globals).flatMap {
+    runtime => jep {
```

Comment:
Minor nit: this is just a `Unit` from `super.injectGlobals`, shouldn't call it `runtime`.

`super.injectGlobals(globals) *> jep { ... }` should do the trick.

Reply:
(the `*>` operator is like `flatMap` but ignores the result of the thing on the left)

Reply:
ok, will change